<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QLiteHtmlSearchWidget</class>
 <widget class="QWidget" name="QLiteHtmlSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>40</height>
   </rect>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QPushButton" name="closeButton">
     <property name="toolTip">
      <string>close search</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="window-close">
       <normaloff>:/media/window-close.svg</normaloff>:/media/window-close.svg</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="searchLabel">
     <property name="text">
      <string>Find:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="searchLineEdit">
     <property name="placeholderText">
      <string>find in text</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="searchDownButton">
     <property name="toolTip">
      <string>search forward</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="go-bottom">
       <normaloff>:/media/go-bottom.svg</normaloff>:/media/go-bottom.svg</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="searchUpButton">
     <property name="toolTip">
      <string>search backward</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="go-top">
       <normaloff>:/media/go-top.svg</normaloff>:/media/go-top.svg</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QComboBox" name="modeComboBox">
     <item>
      <property name="text">
       <string>Plain text</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Whole words</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Regular expression</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="matchCaseSensitiveButton">
     <property name="toolTip">
      <string>Match case sensitive</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="format-text-superscript">
       <normaloff>:/media/format-text-superscript.svg</normaloff>:/media/format-text-superscript.svg</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>searchLineEdit</tabstop>
  <tabstop>searchDownButton</tabstop>
  <tabstop>searchUpButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources>
  <include location="media.qrc"/>
 </resources>
 <connections/>
</ui>
